spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolume do
+  let(:ems) do
+    FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud, :provider_region => "us-south")
+  end
+  let(:StorageManager) { FactoryBot.create(:ems_IbmCloud_StorageManager, :parent_ems_id => ems_cloud.id) }
```

Comment:
This factory needs to be defined in https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/spec/factories/ext_management_system.rb

Let's be consistent with the cloud manager, naming it:
`factory :ems_ibm_cloud_power_virtual_servers_storage`.